### PR TITLE
index.html

### PR DESCRIPTION
**Pull request by ROHIT-ROSHAN**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -115,6 +115,38 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
+
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
Comment (iatharva) [inline index.html:119]: Just replace ' *TEMPLATE: MAKE A COPY*' with your name

File: index.html [hunk 2]
```diff
@@ -115,6 +115,38 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="https://www.coursera.org/in" target="_blank" title="Third resource">
+                  Coursera
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```
Comment (iatharva) [inline index.html:149]: Just replace ' *END TEMPLATE*' with your name

**Review by iatharva — CHANGES_REQUESTED** [review]
Once the changes are done I will merge your pull request. :)

**Comment by ana-vela** [comment]
Hi, @ROHIT-ROSHAN Thank you for submitting this pull request. We are closing pull requests that have been inactive for several months. Please feel free to start over and create a new pull request if you would like to contribute to this project. Thanks!

**Pull request closed without merging**